drivers/wireless/bluetooth/bt_bridge.c

```diff
@@ -69,7 +69,7 @@ struct bt_bridge_s
 #ifdef CONFIG_BLUETOOTH_BRIDGE_BTSNOOP
   FAR struct snoop_s       *snoop;
 #endif /* CONFIG_BLUETOOTH_BRIDGE_BTSNOOP */
-  uint8_t                   refs;
+  atomic_uint               refs;
```

Comment:
Do we need any dependency on atomic config?or atomic_uint is supported unconditionally?

Reply:
atomic_uint is defined by C11, but bt bridge is already guarded by BLUETOOTH_BRIDGE(disabled by default), do we need additional option?

Reply:
Let me take a closer look on Monday